OpenRA.Game/Game.cs

```diff
@@ -776,9 +784,16 @@ static void Loop()
 
 			while (state == RunStatus.Running)
 			{
-				// Ideal time between logic updates. Timestep = 0 means the game is paused
-				// but we still call LogicTick() because it handles pausing internally.
-				var logicInterval = worldRenderer != null && worldRenderer.World.Timestep != 0 ? worldRenderer.World.Timestep : Ui.Timestep;
+				// Ideal time between logic updates.
+				var logicInterval = Ui.Timestep;
+				if (worldRenderer != null)
```

Comment:
The inner `world == null` check can be removed by changing this to `if (worldRenderer?.world != null)`.